plugins/RssWidget/Widgets/RssPiwik.php

```diff
@@ -23,7 +23,7 @@ public static function configure(WidgetConfig $config)
     public function render()
     {
         try {
-            $rss = new RssRenderer('http://feeds.feedburner.com/Piwik');
```

Comment:
In this class RssPiwik we need to apply the same changes as in RssChangelog (https and fallback to http)